modules/ROOT/partials/misc/mobile-platform-compatibility.adoc

```diff
@@ -1,12 +1,39 @@
-== Supported Mobile Platforms
+== Supported Mobile Platforms (1)
 
-{productname} mobile is designed to run on iOS Safari and Android Chrome. {productname} mobile is tested on the following platforms:
+{productname} mobile is tested against, and is designed to run on, the default browser running on a currently vendor-supported version of both Android and iOS.
 
-[cols="^,^,^",options="header"]
+[cols="1,1",options="header"]
 |===
-|OS |Browser |Device
-|Android 11 |Chrome |Mobile phone
-|Android 12 |Chrome |Mobile phone
-|iOS/iPadOS 14 |Safari |iPhone/iPad
-|iOS/iPadOS 15 |Safari |iPhone/iPad
+|Operating System |Default browser
+
+|Android
+|Chrome
+
+|iOS
+|Safari
 |===
+
+Google supports the current and two previous versions of Android.
+
+Apple supports the current and previous versions of iOS.
+
+NOTE: By default, both Android and iOS automatically update both the operating system and default browser. Consequently, {productname} mobile is only supported on the most recent releases of each supported operating system and browser combination.
+
+
+
+== Supported Mobile Platforms (2)
```

Comment:
Why is there a duplicate section?

Reply:
> Why is there a duplicate section

See the [description](https://github.com/tinymce/tinymce-docs/pull/2625#issue-1581238819).